Week1/homework/js-exercises/remainder.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+/* bang operator work by return the reminding after value divided.
+ Ex,if we divide 7 by 3 it will equal 2 and return of the 7 after has been divided 1 */
```

Comment:
Seems like you got the modulo operator! 👍

Reply:
thanks again Sander , looking for the new feedback